polyglot/graalvm/README.md

```diff
@@ -0,0 +1,18 @@
+# Running Polyglot application on Payara Micro with GraalVM
+
+Project contains example of JavaScript library called from Kotlin written JAX-RS resources.
+
+
+       mvn clean package
```

Comment:
I would switch the order of description and command line for all these 3 commands/URLs. It's more natural to have the description first, like:

-----------------------------

Compile project and create WAR package:

```
mvn clean package
```